<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Comic_Creator</class>
 <widget class="QMainWindow" name="Comic_Creator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QSplitter" name="splitter_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QListWidget" name="picView"/>
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QSplitter" name="modSp">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QLabel" name="modL">
             <property name="text">
              <string>模式</string>
             </property>
             <property name="buddy">
              <cstring>ModSelection</cstring>
             </property>
            </widget>
            <widget class="QComboBox" name="ModSelection"/>
           </widget>
          </widget>
         </widget>
         <widget class="QSplitter" name="splitter_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QSplitter" name="metaSp">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QSplitter" name="AuthorSp">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QLabel" name="AuthorL">
              <property name="text">
               <string>作者</string>
              </property>
              <property name="buddy">
               <cstring>editAuthor</cstring>
              </property>
             </widget>
             <widget class="QLineEdit" name="editAuthor"/>
            </widget>
            <widget class="QSplitter" name="BnameSp">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QLabel" name="BnameL">
              <property name="text">
               <string>漫画名</string>
              </property>
              <property name="buddy">
               <cstring>editBname</cstring>
              </property>
             </widget>
             <widget class="QLineEdit" name="editBname"/>
            </widget>
           </widget>
           <widget class="QSplitter" name="PicFileSp">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QPushButton" name="AddPic">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>添加图片文件</string>
             </property>
            </widget>
            <widget class="QPushButton" name="deletePic">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>删除图片文件</string>
             </property>
            </widget>
           </widget>
          </widget>
          <widget class="QSplitter" name="splitter_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QSplitter" name="poSp">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QLabel" name="picOrderL">
             <property name="text">
              <string>图片顺序</string>
             </property>
             <property name="buddy">
              <cstring>picUp</cstring>
             </property>
            </widget>
            <widget class="QSplitter" name="picOrderSp">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QPushButton" name="picUp">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>↑</string>
              </property>
             </widget>
             <widget class="QPushButton" name="picDown">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>↓</string>
              </property>
             </widget>
            </widget>
           </widget>
           <widget class="QPushButton" name="rmFromChap">
            <property name="text">
             <string>从本章中删除</string>
            </property>
           </widget>
          </widget>
          <widget class="QSplitter" name="ChapSp">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QSplitter" name="ChapNameSp">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QLabel" name="chapNameL">
             <property name="text">
              <string>章节名</string>
             </property>
             <property name="buddy">
              <cstring>ChapNameCB</cstring>
             </property>
            </widget>
            <widget class="QComboBox" name="ChapNameCB"/>
            <widget class="QLineEdit" name="editChapName"/>
            <widget class="QPushButton" name="renameChap">
             <property name="text">
              <string>章节重命名</string>
             </property>
            </widget>
           </widget>
           <widget class="QSplitter" name="ChapCtrlSp">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QSplitter" name="chapOderSp">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QPushButton" name="chapUp">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>↑</string>
              </property>
             </widget>
             <widget class="QPushButton" name="chapDown">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>↓</string>
              </property>
             </widget>
            </widget>
            <widget class="QSplitter" name="ChapFileSp">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QPushButton" name="AddChap">
              <property name="text">
               <string>创建新章节</string>
              </property>
             </widget>
             <widget class="QPushButton" name="delChap">
              <property name="text">
               <string>删除章节</string>
              </property>
             </widget>
            </widget>
           </widget>
          </widget>
          <widget class="QSplitter" name="splitter_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QPushButton" name="Clear">
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
           <widget class="QPushButton" name="add2chap">
            <property name="text">
             <string>添加到章节</string>
            </property>
           </widget>
          </widget>
          <widget class="QPushButton" name="generateComic">
           <property name="text">
            <string>生成</string>
           </property>
          </widget>
          <widget class="QSplitter" name="outDirSp">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLabel" name="outputL">
            <property name="text">
             <string>Dircetory</string>
            </property>
           </widget>
           <widget class="QPushButton" name="setOutput">
            <property name="text">
             <string>输出文件夹</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>editBname</tabstop>
  <tabstop>ModSelection</tabstop>
  <tabstop>editAuthor</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
